device/demo/rooms/test/can.esc

```diff
@@ -1,12 +1,46 @@
 :pick_up
-
 set_global i/inv_can true
 set_active can false
 
-
 :look
-
 say player "A can"
 
 :use
 set_state player hat
+
+:open
+set_global turning_can true
```

Comment:
What is this used for?

Reply:
I'll take a look to see if I really need it. I was running into a situation where turning the can evaluated the next block. Naturally you can't call `stop` on it or the player wouldn't move, as `stop` terminates all execution, not just the block.

Any order of the turns will cause two checks of only `[eq can_angle XYZ]` to be true, unless I'm too intellectually challenged to figure out what the order would be. But I'm pretty sure we need this helper boolean. At least it makes the code explicit!

Reply:
I hope this brain game isn't considered a blocker, because I did fidget around with the ordering until I came up with the helper.

Reply:
No, it's not a blocker, but it means I will have to dig into the esc parser again in the future to make this stuff easier. :worried:

Reply:
Mmmmaybe. I never thought I'd hear me say this, but I might actually prefer not having an OR. That would lead to operator precedences and maybe grouping etc. Right now things are simple and doable. Maybe a discussion issue?

If you or anyone can think of an order for these that never breaks, and I'm not saying there isn't one, would be sweet if it could be after merging this :)